### ISSUE TITLE
can not search dictionary

### ISSUE THREAD

**Issue by humorlife** [koreader/kindlepdfviewer#619]
can not search dictionary，suggest add function of dictionary。

**Comment by tigran123** [comment]
Good idea. crengine already has this function, but we'll need to write Lua glue to make use of it (and also provide dictionaries in the format that crengine understands).
